src/python/settings_default.py

```diff
@@ -0,0 +1,15 @@
+# Copy to settings_secret.py, set strict permissions, and modify.
+# DO NOT PUT THE SECRET FILE IN SOURCE CONTROL!
+
+ENDPOINT = "http://localhost:8080/proctor/Services/exams/expire/"
+CLIENT = "SBAC_PT"
```

Comment:
Can we change this to `CLIENT_NAME`?  That will be more consistent with what it is called within SBAC.